### ISSUE TITLE
Fail to replicate tutorial steps

### ISSUE THREAD

**Issue by violafanfani** [biothings/mygene.py#18]
I had some code written for mygene 3.1.0 and it was throwing the "requests.exceptions.HTTPError: 400 Client Error: Expect type list." error. 

I hence reinstalled mygene into a conda environment (mygene=3.2.1, pyh3252c3a_0,bioconda) and tried the steps in the "usage" section of the github readme.

While this query works: 
`mg.getgene(1017)`

I get the following error for the next step:

```
>>> mg.getgene(1017, fields='name,symbol,refseq')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/viola/miniconda3/envs/project-cancer-nets/lib/python3.7/site-packages/biothings_client/base.py", line 321, in _getannotation
    from_cache, ret = self._get(_url, kwargs, none_on_404=True, verbose=verbose)
  File "/home/viola/miniconda3/envs/project-cancer-nets/lib/python3.7/site-packages/biothings_client/base.py", line 163, in _get
    res.raise_for_status()
  File "/home/viola/miniconda3/envs/project-cancer-nets/lib/python3.7/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Expect type list. for url: http://mygene.info/v3/gene/1017
```

What am I missing here? 
Thanks

**Comment by namespacestd0** [comment]
Please try `pip install biothings_client -U` and see https://github.com/biothings/biothings_client.py/issues/17 for more info.

**Comment by violafanfani** [comment]
Ok, it's solved now with biothings_client 0.2.5.
